src/calculation/MeasureReportBuilder.ts

```diff
@@ -647,9 +647,9 @@ export default class MeasureReportBuilder<T extends PopulationGroupResult> exten
     options: CalculationOptions
   ): fhir4.MeasureReport[] {
     const reports: fhir4.MeasureReport[] = [];
+    const measure = extractMeasureFromBundle(measureBundle);
+    const builder = new MeasureReportBuilder(measure, options);
```

Comment:
This line needs to be inside the `forEach` loop below as it needs to be creating a new MeasureReport for each patient.

Good call to move the `extractMeasureFromBundle` outside the loop.

Reply:
whoops! Thank you for the catch!